dev/TreeView/MultiLevelListViewBase.cpp

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "MultiLevelListViewBase.h"
+
+MultiLevelListViewBase::MultiLevelListViewBase(const ITrackerHandleManager* m_owner, const winrt::ListView& lv)
+    : m_viewModel(m_owner), m_listView(m_owner)
+{
+    m_listView.set(lv);
+}
+
+MultiLevelListViewBase::~MultiLevelListViewBase()
+{
+}
+
+com_ptr<ViewModel> MultiLevelListViewBase::ListViewModel() const
+{
+    return m_viewModel.get();
+}
+
+void MultiLevelListViewBase::ListViewModel(com_ptr<ViewModel> viewModel)
+{
+    m_viewModel.set(viewModel);
+}
+
+winrt::TreeViewNode MultiLevelListViewBase::NodeAtFlatIndex(int index) const
+{
+    if (ListViewModel()) {
```

Comment:
>ListViewModel() [](start = 8, length = 15)

prefer auto viewModel = ListViewModel() or auto viewModel = m_viewModel.get(),
and then viewModel->GetNodeAt(index)